azure-iot-provisioning-devicessdk/tests/test_dummy.py

```diff
@@ -0,0 +1,2 @@
+def test_dummy():
```

Comment:
>dummy [](start = 9, length = 5)

nit: dummy will fail the policheck (authorized vocabulary) test. suggest using stub, mock, fake, test... some other word.